app/helpers/rad_common/bootstrap_flash_helper.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
What does this comment mean?

Reply:
Ruby 3 was going to make string literals frozen by default (until Matz decided not to as of 6 months ago, apparently). This 'magic comment' let's us opt into the default frozen string literal functionality on a per-file basis

Reply:
ok